core/src/main/java/com/scalar/db/transaction/consensuscommit/CrudHandler.java

```diff
@@ -51,6 +51,11 @@ public class CrudHandler {
   // Whether the transaction is in read-only mode or not.
   private final boolean readOnly;
 
+  // Whether the transaction is in single-operation mode or not. Single-operation mode refers to
+  // executing a CRUD operation directly through `DistributedTransactionManager` without explicitly
+  // beginning a transaction.
+  private final boolean singleOperation;
```

Comment:
Introduced `single-operation` mode, a mode for transactions executed without explicitly beginning a transaction, such as `DistributedTransactionManager.get()`, `DistributedTransactionManager.scan()`, and `DistributedTransactionManager.insert()`.